source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -68,6 +68,29 @@ bool lldb_private::formatters::swift::SwiftSharedString_SummaryProvider(
       StringPrinter::ReadStringAndDumpToStreamOptions());
 }
 
+// String's Index has the following layout:
+// (mirrored from StringIndex.swift in the standard library.
+//
+// ┌──────────┬───────────────────┬─────────╥────────────────┬──────────┐
+// │ b63:b16  │      b15:b14      │   b13   ║     b12:b8     │  b6:b0   │
+// ├──────────┼───────────────────┼─────────╫────────────────┼──────────┤
+// │ position │ transcoded offset │ aligned ║ grapheme cache │ reserved │
+// └──────────┴───────────────────┴─────────╨────────────────┴──────────┘
```

Comment:
Is there any way around hardcoding this again? This is changing even now: https://github.com/apple/swift/pull/25948

`position` and `transcoded offset` are baked into the ABI, so if you only use those, that's less sketchy

Reply:
Here I only use `offset`, which is part of the ABI, so I guess it should be fine?. I can maybe just point to the structure in the file instead of inlining it here. What do you think?